src/Drinks.js

```diff
@@ -0,0 +1,47 @@
+// Drinks Component
```

Comment:
I liked that when we put the name of a drink like Rum, it gives me results that I can prepare with Rum, very nice